docs/reference/kernel_language.rst

```diff
@@ -327,441 +327,441 @@ Following is the list of supported single precision mathematical functions.
+        | Determine whether argument is a ``NAN``.
       - ✓
       - ✓
 
-    * - | float ldexpf ( float  x, int  exp )
+    * - | ``float ldexpf(float  x, int  exp)``
         | Calculate the value of x ⋅ 2 of the exponent of the input argument.
       - ✓
       - ✓
 
-    * - | loat log10f ( float  x )
+    * - | ``float log10f(float  x)``
         | Calculate the base 10 logarithm of the input argument.
       - ✓
       - ✓
 
-    * - | float log1pf ( float  x )
+    * - | ``float log1pf(float  x)``
         | Calculate the value of the exponent of the input argument
       - ✓
       - ✓
 
-    * - | float logbf ( float  x )
+    * - | ``float logbf(float  x)``
         | Calculate the floating point representation of the exponent of the input argument.
       - ✓
       - ✓
     
-    * - | float log2f ( float  x )
+    * - | ``float log2f(float  x)``
         | Calculate the base 2 logarithm of the input argument.
       - ✓
       - ✓
 
-    * - | float logf ( float  x )
+    * - | ``float logf(float  x)``
         | Calculate the natural logarithm of the input argument.
       - ✓
       - ✓
 
-    * - | float modff ( float  x, float* iptr )
+    * - | ``float modff(float  x, float* iptr)``
         | Break down the input argument into fractional and integral parts.
       - ✓
       - ✗ 
 
-    * - | float nanf ( const char* tagp )
+    * - | ``float nanf(const char* tagp)``
         | Returns "Not a Number" value.
       - ✗ 
       - ✓
 
-    * - | float nearbyintf ( float  x )
+    * - | ``float nearbyintf(float  x)``
         | Round the input argument to the nearest integer.
       - ✓
       - ✓
 
-    * - | float powf ( float  x, float  y )
+    * - | ``float powf(float  x, float  y)``
         | Calculate the value of first argument to the power of second argument.
       - ✓
       - ✓
 
-    * - | float remainderf ( float  x, float  y )
+    * - | ``float remainderf(float  x, float  y)``
         | Compute single-precision floating-point remainder.
       - ✓
       - ✓
 
-    * - | float remquof ( float  x, float  y, int* quo )
+    * - | ``float remquof(float  x, float  y, int* quo)``
         | Compute single-precision floating-point remainder and part of quotient.
       - ✓
       - ✗ 
 
-    * - | float roundf ( float  x )
+    * - | ``float roundf(float  x)``
         | Round to nearest integer value in floating-point.
       - ✓
       - ✓
 
-    * - | float scalbnf ( float  x, int  n )
+    * - | ``float scalbnf(float  x, int  n)``
         | Scale floating-point input by integer power of two.
       - ✓
       - ✓
 
-    * - | __RETURN_TYPE signbit ( float  a )
+    * - | ``bool signbit(float  a)``
         | Return the sign bit of the input.
       - ✓
       - ✓
 
-    * - | void sincosf ( float  x, float* sptr, float* cptr )
+    * - | ``void sincosf(float  x, float* sptr, float* cptr)``
         | Calculate the sine and cosine of the first input argument.
       - ✓
       - ✗ 
```

Comment:
Can you check whether it is supported now?

It seems from this page that it is supported now, but I did not check it.
https://cgmb-hip.readthedocs.io/en/latest/autotoc_md76.html

Thanks!

Reply:
Supported.

Reply:
I have to check this, thanks for the heads up.

Reply:
Fixed.